src/mars_patcher/misc_patches.py

```diff
@@ -70,3 +70,19 @@ def apply_pbs_without_bombs(rom: Rom) -> None:
 
 def apply_anti_softlock_edits(rom: Rom) -> None:
     apply_patch_in_data_path(rom, "anti_softlock.ips")
+
+
+def apply_hint_security(rom: Rom) -> None:
+    rom.write_8(ReservedConstants.HINT_SECURITY_LEVELS_ADDR + 0x4, 0x2)  # S1 unlocked by l2
```

Comment:
I wonder whether we should have a seperate constant for each navigation terminal?
Bio would be able to answer that better than I.